contracts/ownable/assembly/__tests__/index.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { owner, renounceOwnership, transferOwnership } from "..";
```

Comment:
Keeping these tests really simple, because this "contract" is only ever imported and used to augment other contracts. You'll see how this works in a bit.